slashing-protection/src/main/java/tech/pegasys/web3signer/slashingprotection/dao/SignedBlocksDao.java

```diff
@@ -96,4 +96,12 @@ public void insertBlockProposal(final Handle handle, final SignedBlock signedBlo
             "SELECT validator_id, slot, signing_root FROM signed_blocks WHERE validator_id = ?")
         .bind(0, validatorId).mapToBean(SignedBlock.class).stream();
   }
+
+  public Optional<UInt64> maximumSlot(final Handle handle, final int validatorId) {
+    return handle
+        .createQuery("SELECT MAX(slot) FROM signed_blocks WHERE validator_id = ?")
```

Comment:
Can you add a Dao test for the maximumSlot function

Reply:
removed